product-plane-services/policy-server/src/main/java/org/opendatamesh/platform/pp/policy/server/services/PolicyService.java

```diff
@@ -163,6 +165,18 @@ public PolicyResource findPolicyResourceVersion(Long versionId) {
         return mapper.toRes(findPolicyVersion(versionId));
     }
 
+    public List<Policy> findAllFilteredList(PolicySearchOptions filters) {
```

Comment:
questo non serve, puoi direttamente usare quello già presente, passandogli come pageable un PageRequest.of(0, Integer.MAX_VALUE) o un Pageable.unpaged()